MAUI/DataForm/floating-label-layout.md

```diff
@@ -0,0 +1,777 @@
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.ShowLeadingView = true;
+        e.DataFormItem.LeadingView = new Label { Text = "F", FontSize = 18, TextColor = Colors.Gray, FontFamily = "InputLayoutIcons", HeightRequest = 24, VerticalTextAlignment = TextAlignment.End };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+N> The `LeadingView` is applicable for both `Default` and `TextInputLayout` layout types
+
+#### Trailing view
+The trailing view is a view that appears to the right of the text input field. It can be used to add additional functionality or visual elements to the text input, such as a clear button or an icon. It can be added to the floating label layout by setting the `TrailingView` property of [DataFormItem](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormItem.html). You can able show trailing view only when `ShowTrailingView` property is `true.`
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+<dataForm:SfDataForm x:Name="dataForm" 
+                        LayoutType="TextInputLayout"
+                        DataObject="{Binding ContactInfo}">
+<dataForm:SfDataForm.Items>
+    <dataForm:DataFormTextItem FieldName="Name" 
+                                ShowTrailingView="True">
+        <dataForm:DataFormTextItem.TrailingView>
+            <Label Text="F" FontSize="18" TextColor="Gray" FontFamily="InputLayoutIcons" HeightRequest="24" VerticalTextAlignment="End" />
+        </dataForm:DataFormTextItem.TrailingView>
+    </dataForm:DataFormTextItem>
+</dataForm:SfDataForm.Items>
+</dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+    {
+        e.DataFormItem.LayoutType = DataFormLayoutType.TextInputLayout;
+        e.DataFormItem.ShowTrailingView = true;
+        e.DataFormItem.TrailingView = new Label { Text = "F", FontSize = 18, TextColor = Colors.Gray, FontFamily = "InputLayoutIcons", HeightRequest = 24, VerticalTextAlignment = TextAlignment.End };
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+N> The `TrailingView` is applicable only when `LayoutType` is `TextInputLayout`.
+
+#### Leading and Trailing View Position
+The `LeadingViewPosition` and `TrailingViewPosition` properties of `DataFormItem` allow you to adjust the leading and trailing view's positions. The view is positioned `outside` of the container if the leading and trailing view positions are both set to `TextInputLayoutViewPosition.Outside`. The view is positioned `inside` the container if the leading and trailing view positions are both set to `TextInputLayoutViewPosition.Inside`.
```

Comment:
The `LeadingViewPosition` and `TrailingViewPosition` properties of the `DataFormItem` allow you to adjust the leading and trailing view's positions. The view is positioned outside the container if the leading and trailing view positions are set to `TextInputLayoutViewPosition.Outside.` The view is positioned inside the container if the leading and trailing view positions are set to `TextInputLayoutViewPosition.Inside.`

Reply:
Content changed